components/LogSummary.js

```diff
@@ -0,0 +1,132 @@
+import moment from 'moment'
+import { Component } from 'react'
+import fetch from 'isomorphic-unfetch'
+import { withAuth } from 'use-auth0-hooks'
+
+async function secureFetch (url, accessToken, method = 'get', options = {}) {
```

Comment:
This should be moved to a util file and combined with the other secureFetch method in UserList.